docs/en_us/developers/README.rst

```diff
@@ -0,0 +1,45 @@
+######################
+EdX Developer's Guide
+######################
+
+We have moved the edX Developer's Guide to the `edx-documentation`_ repository as of January 13, 2015.
+
+By moving documentation to its own repository, we will be better able to
+develop workflows, manage versioning, create translations, and automate
+testing, without complicating ongoing development of the edX Platform.
+
+.. _edx_documentation: https://github.com/edx/edx-documentation
+
+******************************
+EdX Platform Docstrings
+******************************
+
+We are in the process of creating a new project to publish docstrings for the
+edX Platform. Docstrings will be included for CMS, LMS, and Common modules.
+
+This project will remain in the edX Platform repository, in the ''docs/en-us''
+directory.
```

Comment:
''docs/en-us'' currently has two apostrophe characters on either end instead of quotation marks. Deliberate?